### PR TITLE
[Edit] JavaScript Arrays: .indexOf()

### LINKED ISSUE #1444: [Edit] JavaScript Arrays: `.indexOf()`
### Type of Edit (select all that apply)

Fix grammar, spelling, formatting, Clarify unclear content and/or terms

### Description (optional)

We'd like the following edits to be made to the `.indexOf()` term entry under JavaScript: Arrays:

- Under the "Syntax" section, let's change the code block signature from \```js to \```pseudo.
- For line's # 24-27, can we try to consolidate this information into two list items (each a single, complete sentence)?
- There is a typo on line # 51; can we change from "... after the _first_ element" to "... after the _second_ element"? (emphasis mine)

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by gracekishino**
Fix formatting, Clarify unclear content

### Description

- Fix formatting.
- Consolidate  lines # 24-27 into two list items.
- On line # 51 change from "... after the first element" to "... after the second element".

Closes #1444

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/arrays/terms/indexOf/indexOf.md [hunk 1]
```diff
@@ -48,7 +46,7 @@ console.log(indexOfPineapple);
 // Output: -1
 ```
 
-Check if color `'blue'` is in the `colors` array and after the first element:
+Check if color `'blue'` is in the `colors` array and after the second element:
```
Comment (SSwiniarski) [inline content/javascript/concepts/arrays/terms/indexOf/indexOf.md:49]:
```suggestion
Check if color `'blue'` is in the `colors` array starting with the second element:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@gracekishino I did a first review and left a comment.

File: content/javascript/concepts/arrays/terms/indexOf/indexOf.md (hunk 1 above)
Comment (gracekishino) [reply content/javascript/concepts/arrays/terms/indexOf/indexOf.md:49]: @SSwiniarski thank you! I was trying to think of a way to make that clearer :)

**Review by SSwiniarski — APPROVED** [review]
@gracekishino Looks good for a second review.

File: content/javascript/concepts/arrays/terms/indexOf/indexOf.md [hunk 2]
```diff
@@ -17,18 +17,16 @@ The **`.indexOf()`** method returns the first index at which an element can be f
 
 ## Syntax
 
-```js
+```psuedo
```
Comment (Dusch4593) [inline content/javascript/concepts/arrays/terms/indexOf/indexOf.md:20]:
```suggestion
```pseudo
```

File: content/javascript/concepts/arrays/terms/indexOf/indexOf.md [hunk 3]
```diff
@@ -17,18 +17,16 @@ The **`.indexOf()`** method returns the first index at which an element can be f
...
 array.indexOf(searchElement, startIndex);
```
Comment (Dusch4593) [inline content/javascript/concepts/arrays/terms/indexOf/indexOf.md:21]:
```suggestion
array.indexOf(element, startIndex);
```

File: content/javascript/concepts/arrays/terms/indexOf/indexOf.md [hunk 4]
```diff
@@ -17,18 +17,16 @@ The **`.indexOf()`** method returns the first index at which an element can be f
...
 ```
 
 - `searchElement`: The element we are looking for.
```
Comment (Dusch4593) [inline content/javascript/concepts/arrays/terms/indexOf/indexOf.md:24]:
```suggestion
The following parameters are used in the `.indexOf()` method:

- The `element` to be searched in the `array`.
```

File: content/javascript/concepts/arrays/terms/indexOf/indexOf.md [hunk 5]
```diff
@@ -17,18 +17,16 @@ The **`.indexOf()`** method returns the first index at which an element can be f
...
-- `startIndex`: The index position to begin search from. Optional.
-
-A negative `startIndex` will offset from the end of the array to begin search.
+- `startIndex`: The index position to begin searching from, this is optional. A negative `startIndex` will offset from the end of the array.
```
Comment (Dusch4593) [inline content/javascript/concepts/arrays/terms/indexOf/indexOf.md:25]:
```suggestion
- The optional `startIndex` position to begin searching from. If one is not given, the search starts from the beginning of the array. Negative indices will offset from the end of the `array`.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@gracekishino Just had a few suggestions 😄

**Comment by gracekishino** [comment]
@Dusch4593 thank you for all the suggestions and sorry about the typo for "pseudo"!

I also added "for" in here:
https://github.com/Codecademy/docs/pull/1493/commits/0a39cb2e7d6a8c91f1fc6152d524dc10cf6502a0#diff-ac1ea44d44ca057136986cd745164d7e9e3178c3c587dd66f206f254bb8a5070

**Review by Dusch4593 — APPROVED** [review]
@gracekishino LGTM! 😄

**Comment by Dusch4593** [comment]
Thank you for another wonderful contribution to Codecademy Docs, @gracekishino! 🚀 🎉 🙌🏻 

The changes in the entry should now be live: https://www.codecademy.com/resources/docs/javascript/arrays/indexOf

Screen Shot 2023-01-17 at 10 36 03 PM

Looking forward to your next Docs contribution! 😄

**Pull request merged**